src/app/services/todo.service.ts

```diff
@@ -13,22 +14,26 @@ export class TodoService {
   constructor(private http: HttpClient) { }
 
```

Comment:
consistency in function naming would be nice, you can decde to not have Todo in the function names and I would say its fine since its implementation would be prefaced with "todoService" but then take it off of list and get as well

Reply:
you users function arguably should be in a seperate service regardless if you go with this naming convention